src/objects.js

```diff
@@ -1,41 +1,59 @@
 const createPerson = (name, age) => {
-  // your code here
+return {
+  name: name,
+  age: age,
+
+};
 };
 
 const getName = object => {
-  // your code here
+return object.name;
 };
 
 const getProperty = (property, object) => {
-  // your code here
+return object[property];
 };
 
 const hasProperty = (property, object) => {
-  // your code here
+return object.hasOwnProperty;
 };
```

Comment:
This function doesn't work because `.hasOneProperty` is a method (i.e. requires you to call it by placing `()` after it) that requires you to pass it the argument of the `property`. i.e.
```
const hasProperty = (property, object) => {
  return object.hasOwnProperty(property);
};
```
You can read more on `hasOwnProperty` [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/hasOwnProperty)

Reply:
You can also use the `in` operator to pass this test, i.e.
```
const hasProperty = (property, object) => {
  return property in object;
};
```
You can read more on the `in` operator [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/in)